.eslintrc.js

```diff
@@ -9,19 +9,14 @@ module.exports = {
     'parser': "babel-eslint",
     'parserOptions': {
         "ecmaVersion": 6,
-        "ecmaFeatures": {
-            "jsx": true
-        }
     },
-    "plugins": ["jest", "flowtype", "react", "import", "jsx-a11y"],
+    "plugins": ["jest", "flowtype", "import", "jsx-a11y"],
```

Comment:
I've created another eslint config in kuma/javascript/src that is specific to react, and I've removed the react plugin from this one because it was complaining about not being configured correctly.